glide-core/src/client/mod.rs

```diff
@@ -263,35 +249,31 @@ fn eval_cmd<T: Deref<Target = str>>(hash: &str, keys: Vec<T>, args: Vec<T>) -> C
     cmd
 }
 
-fn to_duration(time_in_millis: u32, default: Duration) -> Duration {
-    if time_in_millis > 0 {
-        Duration::from_millis(time_in_millis as u64)
-    } else {
-        default
-    }
+fn to_duration(time_in_millis: Option<u32>, default: Duration) -> Duration {
+    time_in_millis
+        .map(|val| Duration::from_millis(val as u64))
+        .unwrap_or(default)
 }
 
 async fn create_cluster_client(
     request: ConnectionRequest,
 ) -> RedisResult<redis::cluster_async::ClusterConnection> {
     // TODO - implement timeout for each connection attempt
-    let tls_mode = request.tls_mode.enum_value_or_default();
+    let tls_mode = request.tls_mode.unwrap_or_default();
```

Comment:
No.
1. There is bool switch for TLS https://github.com/aws/glide-for-redis/blob/078d31ed68aa7d52eaf1491ecc203edbe6c930af/glide-core/src/client/mod.rs#L67
2. All clients configure TLS as bool param
3. InsecureTLS is not documented

Reply:
ah, you're talking about enabling insecure TLS in the wrappers. Please, be precise and clear, instead of making vague comments.
We prefer to avoid giving our users APIs which might be misused. Unless you can give me a reasonable scenario that justifies it, the preference is to not let the users use insecure measures.